.github/workflows/build.yml

```
@@ -29,6 +29,12 @@ jobs:
     name: build-example-android
     runs-on: ubuntu-latest # using linux runner for speed
     steps:
+      - name: Set up JDK 17
+        uses: actions/setup-java@v4
+        with:
+          java-version: '17'
+          distribution: 'zulu'
+          cache: 'gradle'
       - name: Checkout
         uses: actions/checkout@v3
```

To fix the build issue on CI, checkout action needs to be triggered first

```suggestion
      - name: Checkout
        uses: actions/checkout@v3
      - name: Set up JDK 17
        uses: actions/setup-java@v4
        with:
          java-version: '17'
          distribution: 'zulu'
          cache: 'gradle'
```